pyproject.toml

```diff
@@ -27,12 +27,11 @@ classifiers = [
 [tool.poetry.scripts]
 ramp = "RAMP.ramp:ramp"
 
-
 [tool.poetry.dependencies]
 python = ">= 3.7,<4"
 click = "^8"
 semantic-version = "^2.8.5"
-redis = "^4.1.0"
+redis = ">= 4.1.0"
```

Comment:
Which did you want, >= 4.1.0 or 5.0.0 as mentioned in the PR? This changes > 4.1.0 to >= 4.1.0 effectively.